tests/compile-fail/double_panic.rs

```diff
@@ -1,4 +1,4 @@
- //error-pattern: the evaluated program aborted
+ //error-pattern: The evaluated program aborted
```

Comment:
The usual style in Rust is for messages to start lower-case, so probably this one should, too.